gem/client/css/themes/malawi.scss

```diff
@@ -132,6 +135,13 @@ a {
       font-family: $font-family-zathu;
       font-size: 12px;
       font-weight: normal;
+      
+      a {
+        font-family: $font-family-default;
+        font-weight: bold;
+        color: $colorOrange;
+        font-size: $font-small;
```

Comment:
looks like it, but its not, first one sits inside  .template-main #article-hero... 😀